### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by calebk1**
This PR is to update the styling of the Swagger docs.

### CODE REVIEW

File: src/swagger-specs/schema-registry.yaml [hunk 1]
```diff
@@ -234,7 +238,7 @@ paths:
     delete:
       tags:
       - "Field groups"
-      summary: Delete a custom field group by ID.
+      summary: Delete a custom field group
       description: >-
         Remove an existing field group through a DELETE request to the `$id` of the field group being removed.
```
Comment (pnolan89) [inline src/swagger-specs/schema-registry.yaml:244]: (247) `deleteFieldGroup`

File: src/swagger-specs/schema-registry.yaml [hunk 2]
```diff
@@ -262,7 +266,7 @@ paths:
     get:
       tags:
       - "Schemas"
-      summary: Retrieve a list of schemas within the specified container.
+      summary: List filtered schemas
       description: >-
         You are able to view a list of all schemas within a container by performing a single GET request. To help filter results, the Schema Registry supports the use of query parameters when listing resources.
```
Comment (pnolan89) [inline src/swagger-specs/schema-registry.yaml:272]: operationId should be just `listSchemas`

File: src/swagger-specs/schema-registry.yaml [hunk 3]
```diff
@@ -436,7 +442,7 @@ paths:
     get:
       tags:
         - "Behaviors"
-      summary: Retrieve a list of behaviors.
+      summary: List filtered behaviors
       produces:
         - application/json
```
Comment (pnolan89) [inline src/swagger-specs/schema-registry.yaml:447]: Missing operationId

File: src/swagger-specs/schema-registry.yaml [hunk 4]
```diff
@@ -488,7 +494,7 @@ paths:
     get:
       tags:
         - "Behaviors"
-      summary: Retrieve a behavior by ID.
+      summary: Retrieve a behavior
       produces:
         - application/json
       parameters:
```
Comment (pnolan89) [inline src/swagger-specs/schema-registry.yaml:500]: Missing operationId

File: src/swagger-specs/schema-registry.yaml [hunk 5]
```diff
@@ -556,9 +562,9 @@ paths:
     get:
       tags:
       - "Mixins (deprecated)"
-      summary: Retrieve a list of mixins within the specified container.
+      summary: List all mixins
```
Comment (pnolan89) [inline src/swagger-specs/schema-registry.yaml:565]: "List mixins"

File: src/swagger-specs/schema-registry.yaml [hunk 6]
```diff
@@ -593,7 +599,7 @@ paths:
       - $ref: '#/parameters/x-gw-ims-org-id'
       - $ref: '#/parameters/x-sandbox-name'
       - $ref: '#/parameters/accept'
-      summary: Retrieve a mixin within the specified container by ID.
+      summary: Retrieve a mixin
       deprecated: true
       operationId: lookupMixin
```
Comment (pnolan89) [inline src/swagger-specs/schema-registry.yaml:604]: `retrieveMixin`

File: src/swagger-specs/schema-registry.yaml [hunk 7]
```diff
@@ -640,7 +646,7 @@ paths:
     put:
       tags:
       - "Mixins (deprecated)"
-      summary: Update a custom mixin by ID.
+      summary: Update a custom mixin
       deprecated: true
       description: A PUT request is essentially re-writing the mixin, therefore the request body must include all fields required to create (POST) a mixin. This is especially useful when updating a lot of information in the mixin at once.
       operationId: replaceMixin
```
Comment (pnolan89) [inline src/swagger-specs/schema-registry.yaml:652]: `updateMixin`

File: src/swagger-specs/audit-query.yaml [hunk 8]
```diff
@@ -41,7 +41,7 @@ paths:
     get:
       tags:
       - Audit events
-      summary: Retrieve a list of audit events
+      summary: List filtered audit events
```
[the remark this answers is not in the document]
Comment (calebk1) [reply src/swagger-specs/audit-query.yaml:44]: Oh - this is the thing I thought we were agreed on for "List" - but I'll go fix those up then!

File: src/swagger-specs/dataset-service.yaml [hunk 9]
```diff
@@ -36,8 +36,9 @@ paths:
     get:
       tags:
         - Datasets
-      summary: Retrieve data usage labels for a specified dataset
-      operationId: getDatasetLabels
+      summary: Retrieve data usage labels 
+      description: You can use this endpoint to retrieve data usage labels for a specified dataset.
```
[the remark this answers is not in the document]
Comment (calebk1) [reply src/swagger-specs/dataset-service.yaml:40]: 👍

**Pull request merged**